packages/puppeteer/src/legacy.ts

```diff
@@ -51,9 +52,8 @@ export async function injectJS(
 }
 
 async function injectJSModule(frame: Frame): Promise<void> {
-  await frame.addScriptTag({
-    path: require.resolve('axe-core')
-  });
+  const source = fs.readFileSync(require.resolve('axe-core'), 'utf8');
+  await injectJSSource(frame, source);
```

Comment:
I had to make this change, because apparently `addScriptTag` changes the source of the page, which caused problems with the `html` property on results.